README.md

```diff
@@ -107,22 +120,72 @@ depending on the provided *Content-Type* header.
     query string, which provides the `query` parameter.
 
 
-### Advanced Options
+## Preparing a Document
+In order to avoid malicious attacks in production, it is recommended to only
+allow approved GraphQL queries to be completely certain none of them will drain
+backend resources in an unintended fashion. Therefore this GraphQL middleware
```

Comment:
Going to remove this in a merge, since that PR is still under discussion. I think this language is too strong, I would personally recommend against this approach for dealing with malicious attacks as it removes one of the primary benefits of GraphQL: that existing clients will not break when servers evolve.